emails/tests/fixtures/encoded_trailing_newline_incoming.email

```diff
@@ -0,0 +1,17 @@
+Date: Tue, 09 Jul 2024 20:02:30 +0200
+Subject: An =?UTF-8?Q?encoded_newline=0A?=

```

Comment:
praise: thank you for plumbing the depths of email content down to the character level!